docs/framework/xaml-services/x-static-markup-extension.md

```diff
@@ -53,7 +53,7 @@ References any static by-value code entity that is defined in a [!INCLUDE[TLA#tl
   
  **Object element syntax:** `<x:Static Member="` `prefix` `:` `typeName` `.` `staticMemberName` `" .../>`  
   
- **Attribute syntax with explicit Member property for initialization string:** `<` `object` `` `property` `="{x:Static Member=` `prefix` `:` `typeName` `.` `staticMemberName` `}" .../>`  
+ **Attribute syntax with explicit Member property for initialization string:** `<` `object` `property` `="{x:Static Member=` `prefix` `:` `typeName` `.` `staticMemberName` `}" .../>`  
```

Comment:
the reason why this and the previous one is all broken down is because previously on MSDN we'd use different elements to produce this syntax block:
![image](https://user-images.githubusercontent.com/12971179/43597195-329e2fc6-9636-11e8-86c9-24c8ddc36255.png)

We don't have this ability with docs/Markdown, so my recommendation would be to put this all in one inline block element instead of breaking it down like this

Reply:
I'd put it in on a separate fenced line. The "Attribute syntax" breaks in the original Markdlg page, which I think is confusing.